README.rst

```diff
@@ -67,6 +67,87 @@ If you want to build the documentation exactly as seen on the website, use
 ``make py`` followed by ``make htmlall``. This builds some
 external dependencies and all supported languages.
 
+Use virtualenv
+==============
+
+To develop documentation in a ``virtualenv``, use the ``venv`` target.
+Clear Linux\* OS documentation provides a make target ``venv`` as a 
+simple way for you to assure that your development environment has the 
+latest packages and that you manage Python versions separately. Use of the 
+``virtualenv`` requires **Python 3.6** or higher. For Windows examples below, use Powershell as an Administrator.
+
+The version of Python used to **create the virtual environment** is the 
+default python version **used in the virtual environment**. 
+
+#. Verify ``pip`` is installed. A file path to pip should appear. 
+
+   On |CL| and macOS\*:
+
+   .. code-block:: bash
+
+      which pip
+   
+   On Windows\* 10 OS: 
+
+   .. code-block:: bash
+
+      pip --version
+
+#. If ``pip`` is not installed, install it. 
+
+   On |CL| and macOS\*:
+
+   .. code-block:: bash
+
+      python3 -m pip install --user --upgrade pip
+
+   On Windows\* 10 OS: 
+
+   .. code-block:: bash
+
+      py -m pip install --upgrade pip
+
+   .. note::
+
+      This assumes Python was already added to your Windows path. 
+
+Install virtualenv 
+------------------
+
+#. Install ``virtualenv``. 
+
+   * On Clear Linux* OS and macOS*:
```

Comment:
Should this line read `* On |CL| and macOS\*:`, like it was written in the previous section?

Reply:
I realized that all mentions of Clear Linux OS need to be consistently spelled out--because the README is outside scope of  `source` folder .